cleanup.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
```

Comment:
I would prefer to drop the cleanup.sh completely, because the functionality is now covered by the maintainer-clean target. If we keep it, we should copy the header from bootstrap.sh.

Reply:
Not sure if anyone is using it, so better keep it. And yes, copy the header from bootstrap.sh (with updated copyright year maybe).

Reply:
ok, I'll add the header in patch 8/9 and push it again.